src/pages/iou/WaypointEditor.js

```diff
@@ -134,7 +154,6 @@ function WaypointEditor({transactionID, route: {params: {iouType = '', waypointI
                     <AddressSearch
                         inputID={`waypoint${waypointIndex}`}
                         ref={(e) => (textInput.current = e)}
-                        hint={!network.isOffline ? translate('distance.errors.selectSuggestedAddress') : ''}
```

Comment:
@huzaifa-99 

This error is actually shown if you don't select an address from the suggestion and then try to save the address.

This is because the location coordinates are tied to each location suggestion, without clicking a suggestion, we don't have the coordinates. So we need the user to click a suggestion.

Are you getting this error when using the user's current location?

Reply:
ah i see can you add "or your current location"

Reply:
and maybe we have to request the spansish translation again 😓

Reply:
I think we can just treat the text input hint and the button separately

so...

```
Please selected a suggested address or use current location
Use current location
```